CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Constellation Changes
 
+## Changes in June 2021
+
+-    `ImportTableColumn.validate` and `importexport/RunPane.validate` 
+	functions return type changed from `void` to `boolean`.
```

Comment:
If you can slightly rephrase this to start with "Changed" or "Updated" (just to align it with the way other entries are written), that would be great